### DISCUSSION TITLE
RAG upload file type support - Powerpoint ppt/pptx

### DISCUSSION THREAD

**Discussion by ips972** [open-webui/open-webui#7860] [Ideas]
looking over at the code, i found that only word and excel are supported file types for RAG upload. 
also seeing the error in docker logs:
INFO  [open_webui.apps.webui.routers.files] file.content_type: application/vnd.openxmlformats-officedocument.presentationml.presentation
ERROR [open_webui.apps.retrieval.main] HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "/app/backend/open_webui/apps/retrieval/main.py", line 960, in process_file
    docs = loader.load(
           ^^^^^^^^^^^^

would you be so kind to add these file types too: ppt, pptx. 

also - please note that older word, doc extentions are causing tracebacks and unsupported to. had to convert all of them to docx or pdf for upload.

**Comment by ips972** [comment]
any update on this?

**Comment by iunknown80** [comment]
PowerPoint is supported from version [0.1.125] - 2024-05-19
